test/with_dummyserver/test_https.py

```diff
@@ -244,7 +244,7 @@ def test_verified(self) -> None:
                 r = https_pool.request("GET", "/")
                 assert r.status == 200
 
-            assert w == []
+            assert [str(wm) for wm in w] == []
```

Comment:
Oh, did you use that to help you debug? Because the two statements are equivalent otherwise